src/enumerate.js

```diff
@@ -1,66 +0,0 @@
-/**
- * @constructor
- * @param {Object} iterable
- * @param {number=} start
- * @extends Sk.builtin.object
- */
-Sk.builtin.enumerate = function (iterable, start) {
```

Comment:
You found it. But yes I put these all in iterator_objects.js. Mostly because when I was working on the interface they were so similar. I can put things back into their own files if preferred.